lxqt-sudo: lxsu & lxsudo symlinks for lxqt-sudo

palinek opened the pull request:
closes lxde/lxqt#794

jleclanche commented:
Why `-f`?

palinek commented:
> Why -f?

The symlink could exist from previous build...

paulolieuthier commented on sudo.cpp:
```diff
@@ -1,116 +1,269 @@
- * Boston, MA 02110-1301 USA
- *
- * END_COMMON_COPYRIGHT_HEADER */
 
```
No license?

palinek replied:
That's a mistake, of course.

paulolieuthier commented:
@palinek GTM. Please squash/rename the commits as you with and ship it. @pmattern I think a issue must be files in pcmanfm-qt's bugtracker.

palinek commented:
@paulolieuthier actually there is a discussion in lxde/lxqt#794 about the naming etc.... and I'm confused if the lxsudo/lxsu is wanted.

palinek commented:
@pmattern with @3471832 lxqt-sudo supports multiple arguments for su backend... so no need for quotting in pcmanfm-qt configuration.

paulolieuthier commented:
> Are you sure about this? I moved the argument size check after "options removal" intentionally. It's a check if a command to run was provided.

Sorry, I was just trying to fix a segfault.

> @paulolieuthier actually there is a discussion in lxde/lxqt#794 about the naming etc.... and I'm confused if the lxsudo/lxsu is wanted.

Yeah, I'm trying to end that discussion. Apparently, it has come to some agreement.

palinek commented:
> > Are you sure about this? I moved the argument size check after "options removal" intentionally. It's a check if a command to run was provided.

> Sorry, I was just trying to fix a segfault.

I see it now.... updated.
